identity/app/services/ExactTargetService.scala

```diff
@@ -0,0 +1,44 @@
+package services
+
+import com.exacttarget.fuelsdk.{ETResponse, ETSubscriber}
+import utils.SafeLogging
+import common.ExecutionContexts
+import conf.IdentityConfiguration
+import scala.concurrent.Future
+import scalaz.\/
```

Comment:
Ah, just realised I'm wrong here. You're using scalaz's EitherT elsewhere so the consistency argument doesn't apply. I'd very strongly favour cats over scalaz for these use cases but if you're doing that for consistency with existing code then that'd be an argument for using scalaz.